fixed_frame: velodyne    # rviz中的fixed frame
R          : 60          # 极坐标栅格地图半径上的分割数
TH         : 180         # 极坐标栅格地图角度上的分割数
grid_size_r: 0.4         # 极坐标栅格地图半径上的分辨率
grid_size  : 0.2         # 输出栅格大小
threshold  : 0.15        # 地面点阈值
cut_x      : 1.5         # 裁剪近处自车点x范围
cut_y      : 0.8         # 裁剪近处自车点y范围
y_width    : 50          # 输出栅格地图一侧宽度
x_forward  : 100         # 输出栅格地图前向长度
RANSAC_threshold: 0.2    # RANSAC 阈值